Lib/test/test_itertools.py

```diff
@@ -1821,6 +1821,13 @@ def test_zip_longest_result_gc(self):
         gc.collect()
         self.assertTrue(gc.is_tracked(next(it)))
 
+    @support.cpython_only
+    def test_pairwise_result_gc(self):
+        # Ditto for pairwise.
+        it = pairwise([None, None])
+        gc.collect()
```

Comment:
In all tests above except for `zip_longest` there is a `next(it)` before `gc.collect()`. All these tests were added in 226a012d1cd61f42ecd3056c554922f359a1a35d (bpo-42536).

@brandtbucher, why is such difference? Is there a bug in `test_zip_longest_result_gc`? Do we need `next(it)` there and here?